contracts/payloads/IGP107/PayloadIGP107.sol

```diff
@@ -0,0 +1,487 @@
+
+    function _PROPOSAL_ID() internal view override returns (uint256) {
+        return PROPOSAL_ID;
+    }
+
+    /**
+     * |
+     * |     Proposal Payload Actions      |
+     * |__________________________________
+     */
+
+    // @notice Action 1: Withdraw ETH for Solana LP and rewards
+    function action1() internal isActionSkippable(1) {
+        string[] memory targets = new string[](1);
+        bytes[] memory encodedSpells = new bytes[](1);
+
+        string
+            memory withdrawSignature = "withdraw(address,uint256,address,uint256,uint256)";
+
+        // Spell 1: Transfer ETH to Team Multisig for Solana LP and rewards
+        {
+            uint256 ETH_AMOUNT = 230 * 1e18; // 230 ETH
+            targets[0] = "BASIC-A";
+            encodedSpells[0] = abi.encodeWithSignature(
+                withdrawSignature,
+                ETH_ADDRESS,
+                ETH_AMOUNT,
+                TEAM_MULTISIG,
+                0,
+                0
+            );
+        }
+
+        IDSAV2(TREASURY).cast(targets, encodedSpells, address(this));
+    }
+
+     // @notice Action 2: Provide Credit to Team Multisig for Fluid DEX Lite
+    function action2() internal isActionSkippable(2) {
+        // Give Team Multisig 4M USDC credit
+        {
+            FluidLiquidityAdminStructs.UserBorrowConfig[]
+                memory configs_ = new FluidLiquidityAdminStructs.UserBorrowConfig[](
+                    1
+                );
+
+            configs_[0] = FluidLiquidityAdminStructs.UserBorrowConfig({
+                user: TEAM_MULTISIG,
+                token: USDC_ADDRESS,
+                mode: 1,
+                expandPercent: 1 * 1e2, // 1%
+                expandDuration: 16777215, // max time
+                baseDebtCeiling: getRawAmount(
+                    USDC_ADDRESS,
+                    0,
+                    4_000_000, // $4M -> additional 2M
+                    false
+                ),
+                maxDebtCeiling: getRawAmount(
+                    USDC_ADDRESS,
+                    0,
+                    4_000_000, // $4M -> additional 2M
+                    false
+                )
+            });
+
+            LIQUIDITY.updateUserBorrowConfigs(configs_);
+        }
+
+        // Give Team Multisig 4M USDT credit
+        {
+            FluidLiquidityAdminStructs.UserBorrowConfig[]
+                memory configs_ = new FluidLiquidityAdminStructs.UserBorrowConfig[](
+                    1
+                );
+
+            configs_[0] = FluidLiquidityAdminStructs.UserBorrowConfig({
+                user: TEAM_MULTISIG,
+                token: USDT_ADDRESS,
+                mode: 1,
+                expandPercent: 1 * 1e2, // 1%
+                expandDuration: 16777215, // max time
+                baseDebtCeiling: getRawAmount(
+                    USDT_ADDRESS,
+                    0,
+                    6_000_000, // $6M -> additional 2M each for USDC/USDT and USDe/USDT
```

Comment:
we set 2.5M USDC and 2.5M USDT in the [IGP105](https://github.com/Instadapp/fluid-governance/blob/7e1a05c9f90b1b18ce7df1f12db36a1ba7dd86d0/contracts/payloads/IGP105/PayloadIGP105.sol#L467), so should set new values accordingly. one time 4M for USDC and one time 6M for USDT seems off?